pkg/settings/setting.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // MaxSettings is the maximum number of settings that the system supports.
 // Exported for tests.
-const MaxSettings = 512
+const MaxSettings = 288
```

Comment:
Guessing we don't want to decrease this :D

Reply:
is this persisted somewhere?

Reply:
Not persisted, but if we register more settings than the max a panic is thrown.  My guess is a few PRs bumped it when we hit the old limit, so we should probably stick with the higher value.

Reply:
Fixed. Thanks.

Reply:
Removed one of the commits since the setting was increased multiple times.